app/models/chef.rb

```diff
@@ -1,4 +1,8 @@
 class Chef < ApplicationRecord
    validates_presence_of :name
    has_many :dishes
+
+   def ingredients_used
+     Ingredient.joins(dishes: :chef).where(chefs: { id: id })
```

Comment:
This query might have some duplicates so you'd want to add `distinct`

Also, you can leverage the associations that are set up in the dishes model since this is the parent object. You could add to Chef this association: `has_many :ingredients, through: :dishes` and then that. allows you to do `chef.ingredients.distinct`